### ISSUE TITLE
Fields have empty values

### ISSUE THREAD

**Issue by morgannilsson** [pimcore/generic-data-index-bundle#163]
We use Pimcore 2024.1 with "pimcore/generic-data-index-bundle": v1.0.5
We use "pimcore/customer-management-framework-bundle": v4.0.6

Some fields are empty in the index, e.g.

```json
"standard_fields": {
	"email": null,
	"firstname": null,
	"firstNames": "Per Mikael",
	"lastname": null,
	"fullName": "Per Mikael Lind"
}
```

In the class:
email (Type: Email)
firstname (Type: Firstname)
firstNames (Type: Input)
lastname (Type: Lastname)
fullName (Type: Input)

The Object Person contains data in these fields.

Are there any plans to implements these types?

**Comment by markus-moser** [comment]
Thanks for your input. Yes here are special CRM related field types in use which seem to be missing at the moment. It should be relatively easy to support them and we will add the support soon 👍

**Comment by markus-moser** [comment]
There are also some CRM related field types in other bundles - I created issues for them:

https://github.com/pimcore/personalization-bundle/issues/76
https://github.com/pimcore/newsletter-bundle/issues/52

**Comment by markus-moser** [comment]
The CRM related field definition adapters from the core are already implemented in version 1.1.0 of the generic data index bundle which will be released soon - see https://github.com/pimcore/generic-data-index-bundle/pull/125
